notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -10483,6 +10483,64 @@ drug abuse [Linda was abusing alcohol and the happiness-producing drug Soma.]
+alcohol abuse [The viewer learns that Mrs. Rittenhouse had taken to the bottle in the wake of her husband having a series of affairs, and had subsequently "stepped it up last summer" after he son died of leukemia.]
+extramarital affair [Mrs. Rittenhouse met with the MSNBC reporter Jenny to give her the name of the woman that Mrs. Rittenhouse's husband was supposedly having an affair with.]
+mother and daughter [Jenny comforted her forgetful, rambling, down in the dumps mother, Robin.]
+coping with getting dumped [Robin didn't take it well when her husband left her for a much younger woman.]
+husband and wife [Jason and his conspicuously young new wife. Don and Ellen Beiderman. Chuck and Wendy Hotchner. Leo married Sarah in a desperate effort to make so that she's be able to join him in the safety of the government cave shelter by the time the comet hit Earth.]
+government secrecy [It was revealed that the United States and Russia had kept secret from the public for 8 months that a massive comet was on a collision course with Earth.]
+nuclear weapons [The Messiah crew used nuclear bombs on the comet.]
+mass hysteria [President Beck explained that having revealed to the public that a massive comet was on a collision course with Earth before a plan was in place to deflect would have only resulted in panic in the streets. Later, news footage from Moscow showed a chaotic scene of people doing just that.]
+young character vs. old character [The young, up and coming astronaut Oren Monash openly expressed his conviction that the veteran astronaut Captain Tanner's was too old and over the hill to be useful to the mission, but the resourceful old captain proved him wrong, and they came to mutually respect one another.]
+literature appreciation [Captain Tanner spoke of Melville and Twain to crewman Monash.]
+male bonding [After getting off on the wrong foot, Captain Tanner bonded with crewman Monash by speaking to him about Melville and Twain while Monash lay in sickbay.]
+human tough decision [Should Sarah go to the government cave to take shelter from the comet strike or die together with her loving parents?]
+human self-sacrifice to prevent mass death [The Messiah crew blew themselves up in a desperate bid to prevent a gigantic comet from hitting the Earth.]
+parental love [Robin pointedly told her teenage daughter that she loved her, and urged her to go to the government shelter and save herself.]
+saying goodbye forever [The Messiah crew members bid their loved ones farewell shortly before blowing themselves up in a last ditched effort to prevent the comet from striking Earth.]
+rebuilding society after a disaster [The film concludes with President Beck giving a speech about how the world will rebuild from a disastrous cometary impact that killed millions and left vast swaths of North America, Africa, and Europe utterly destroyed.]
+coping with a disability [Crewman Monash was blinded while rigging nuclear bombs on the comet.]
```

Comment:
Was he seen coping with this later though, I find myself wondering?

Reply:
Very minorly so, I'd say. He had to ask if it was his wife on a video feed or something along those lines.